oxoPattern1.java

```diff
@@ -0,0 +1,28 @@
+import java.util.InputMismatchException;
+import java.util.Scanner;
+public class oxoPattern1 {
+    public static void main(String[] args) {
+        Scanner userInputScanner = new Scanner(System.in);
+        try{
+            System.out.print("Line:");
+            int userInputLines = userInputScanner.nextInt();
+            System.out.print("Column:");
+            int userInputColumns = userInputScanner.nextInt();
+            for(int i = 1; i <= userInputLines; i++){
+                if(i % 2 == 0){
+                    for(int j = 1; j <= userInputColumns; j++){
+                        System.out.print("xo");
+                    }
+                } else {
```

Comment:
instead of 2 loops use variable set to change what to write and use 1 loop with this variable